tasks/src/video-classification/data.ts

```diff
@@ -66,7 +66,12 @@ const taskData: TaskDataCustom = {
 			id:          "microsoft/xclip-base-patch32",
 		},
 	],
-	spaces:       [],
+	spaces:       [
+		{
+			description: "An application that classifies video at different timestamps.",
```

Comment:
https://huggingface.co/spaces/fcakyon/video-classification as well likely, as this one uses `transformers`